CHANGELOG.md

```
@@ -6,6 +6,12 @@
 - feat: provide collection functionality by @Rindrics in https://github.com/Rindrics/izuminokami-kanesada/pull/128
 - feat: add lunyu contents by @Rindrics in https://github.com/Rindrics/izuminokami-kanesada/pull/130
 
+## [v0.7.0](https://github.com/Rindrics/izuminokami-kanesada/compare/v0.6.0...v0.7.0) - 2026-02-12
```

markdownlint-cli2 (0.20.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)